setup.py

```diff
@@ -1,14 +1,15 @@
 # -*- coding: utf-8 -*-
 """Setup module."""
 try:
-    from setuptools import setup
+    from setuptools import setup, find_packages
 except ImportError:
     from distutils.core import setup
 
 
-def get_requires():
+def get_requires(streaming=False):
     """Read requirements.txt."""
-    requirements = open("requirements.txt", "r").read()
+    reqs = "requirements.txt" if not streaming else "streaming-requirements.txt"
+    requirements = open(reqs, "r").read()
```

Comment:
This looks like a hot fix, how about a dictionary that maps `streaming` and `core` to `streaming-requirements.txt` and `requirements.txt`.

Reply:
I know your taste :), sure I will do.

Reply:
applied